modules/pipeline/precheck/checkers/diceymlchecker/diceyml.go

```diff
@@ -74,7 +74,7 @@ func checkDiceYmlAndDiceWorkspaceYml(p *spec.Pipeline) error {
 	diceworkspaceymlName = fmt.Sprintf("dice_%s.yml", diceymlworkspace)
 
 	repo := gittarutil.NewRepo(discover.Gittar(), p.CommitDetail.RepoAbbr)
-	diceymlcontent, err := repo.FetchFile(p.GetCommitID(), diceymlName)
+	diceymlcontent, err := repo.FetchFile(p.GetCommitID(), diceymlName, p.GetRunUserID())
```

Comment:
precheck will be invoked when pipeline created.
So runUserID always be empty.

Reply:
Add a function called GetUserID, get users by runUserID/submitUserID in order.

Reply:
done